@shared/api/internal.ts

```diff
@@ -1529,3 +1543,28 @@ export const simulateTransaction = async (args: {
     response,
   };
 };
+
+export const getIsAccountMismatch = async ({
+  activePublicKey,
+}: {
+  activePublicKey: string;
+}) => {
+  let response = {
+    error: "",
+    isAccountMismatch: false,
+  };
+
+  response = await sendMessageToBackground({
+    type: SERVICE_TYPES.GET_IS_ACCOUNT_MISMATCH,
+    activePublicKey,
+  });
+
+  if (response.error) {
+    throw new Error(response.error);
+  }
+
+  return {
+    isAccountMismatch: response.isAccountMismatch,
+    error: response.error,
+  };
```

Comment:
this duck will be used to check account mismatch status and set the redux state accordingly